oci-artificial-intelligence/ai-speech/customizations/customizations.md

```diff
@@ -103,11 +103,162 @@ To view your newly created customization, click on your customization's name fro
+            },{
+                    "entityType": "bird",
+                    "entities": [{
+                            "entityValue": "Tweeteee3",
+                            "pronunciations": [{
+                                    "soundsLike": "tweetyy"
+                                }
+                            ],
+                            "weight": 1
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+    </copy>
+    ```
+    *Note:*
+    * Supported values for domain are GENERIC, and MEDICAL
+    * Supported values for languageCodePLaceHolder are en-US, en-AU, en-IN, en-GB, it-IT, pt-BR, hi-IN, fr-FR, de-DE, es-ES
+
+
+2. <u>Get Customization Job</u>
+    Endpoint:
+    ```
+    <copy> 
+        https://speech.aiservice.us-phoenix-1.oci.oraclecloud.com/customizations/ocid1.aispeechcustomization.dev.dev
+    </copy>
+    ```
+    `ocid1.aispeechcustomization.dev.dev` should be replaced with the customization ID/Alias
+
+3. <u>List Customization Jobs</u>
+    Endpoint:
+    ```
+    <copy> 
+        https://speech.aiservice.us-phoenix-1.oci.oraclecloud.com/20220101/customizations?compartmentId=ocid1.compartment.oc1..<uniqueId>
+    </copy>
+    ```
+    `ocid1.compartment.oc1..<uniqueId>` should be replaced with the compartment ID 
+4. <u>Update Customization Job</u>
+    ![Update customization job](./images/customization-rest-update.png)
+
+    With the updated Customization body. We can submit a PUT request to update the Customization
+    
+    Endpoint:
+    ```
+    <copy>
+    https://speech.aiservice.<region-identifier>.oci.oraclecloud.com/20220101/customizations?
+    </copy>
+    ```
+
+    Body:
+    ```
+    <copy>
+    {
+    "compartmentId": "ocid1.compartment.oc1..<uniqueId>",
+    "description": "<descriptionPlaceholder>",
+    "displayName": "<displayNamePlaceholder>",
+    "alias": "<aliasPlaceHolderNew>",
+    "modelDetails": {
+        "domain": "GENERIC",
+        "languageCode": "en-US"
+    },
+    "trainingDataset": {
+        "datasetType": "ENTITY_LIST",
+        "referenceExamples": [
+            "Private data is <private>",
+            "Bird name is <bird>"
+        ],
+        "entityList": [{
+            "entityType": "private",
+            "entities": [{
+                    "entityValue": "hello",
+                    "pronunciations": [{
+                            "soundsLike": "helloo"
+                        }
+                    ],
+                    "weight": 1
+                }
+            ]
+        },{
+                "entityType": "bird",
+                "entities": [{
+                        "entityValue": "Robin",
+                        "pronunciations": [{
+                                "soundsLike": "Robun"
+                            }
+                        ],
+                        "weight": 1
+                    }
+                ]
+            }
+        ]
+    }
+}
+
+    </copy>
+    ```
+
```

Comment:
Missing delete customization endpoint,
also make sure to include the type of request that each operation is (GET, PUT, UPDATE, DELETE etc) along with the endpoint